### PR TITLE
polished acct page

### PR DESCRIPTION
**Pull request by JoeWrightNC**
account page made nice and clean, moved some stuff into partials to be able to call it there, we can cut down on code by calling those partials on event and resource post pages if you all want, update info route and contact page coming very soon

### CODE REVIEW

File: public/assets/js/profile-controller.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+$("#profileUpdate").on("click", function() {
+  $("#profileEventForm").hide()
```
Comment (schmitty890) [inline public/assets/js/profile-controller.js]: if this was annoying to write out .hide() over and over again, an alternative could be comma separating these.

$('#profileEventForm, #profileClassForm, #profileResForm, #profileContactForm').hide();

does the same outcome, it just selects all those elements at once, and hides, vs technically selecting one, then hide, then the next, then hide, and so on.

**Comment by JoeWrightNC** [comment]
@schmitty890 that makes sense, I didn't think of it before the fact.  I figured the other option was to link out to those pages and leave the account page, but I thought this looked much slicker and was a better ui/ux.  can we give them more specific IDs where needed (i.e. will that work with datepicker infrastructure)?

**Comment by JoeWrightNC** [comment]
@schmitty890 also, Could we potentially just get rid of the datepicker on the resources form?  Not sure it is really needed there anyway right?

**Comment by tommyngre** [comment]
@JoeWrightNC @schmitty890 My only thought with the Datepicker on the resource page was for use as an "expiration date." I wouldn't miss it.

**Comment by schmitty890** [comment]
@JoeWrightNC  @tommyngre  we could strip it out. we could also switch it to target any element with a class of 'date' instead of an id of 'date'. then that datepicker would work as expected. we'll refine these little issues over the next few days.

**Pull request merged**